koku/api/settings/tags/mapping/utils.py

```diff
@@ -23,6 +28,36 @@
 from reporting.provider.ocp.models import OCPUsageReportPeriod
 
 
+class TagMappingFilters(SettingsFilter):
+    """This filter handles multiple filter options."""
+
+    def _combine_query_filters(self, q_list: List[Q]) -> Q:
```

Comment:
`typing.List` is a deprecated alias to `list`.
```suggestion
    def _combine_query_filters(self, q_list: list[Q]) -> Q:
```